lib/explorer/backend/lazy_frame.ex

```diff
@@ -73,10 +74,21 @@ defmodule Explorer.Backend.LazyFrame do
   @impl true
   def pull(df, column) do
     dtype_for_column = df.dtypes[column]
+    series_backend = get_series_backend(df.data.original_data)
+
     data = LazySeries.new(:column, [column], dtype_for_column)
+    data = %{data | backend: series_backend}
+
     Backend.Series.new(data, dtype_for_column)
   end
 
+  defp get_series_backend(%module{}) do
+    module
+    |> Module.split()
+    |> List.replace_at(-1, "Series")
+    |> Module.concat()
```

Comment:
I am not sure we can assume they have related names. So I would pass the dataframe backend instead and make it a dataframe callback?

Reply:
Yeah, makes sense!